src/internal.c

```diff
@@ -7050,30 +7050,10 @@ static int BuildSHA(WOLFSSL* ssl, Hashes* hashes, const byte* sender)
 static int BuildFinished(WOLFSSL* ssl, Hashes* hashes, const byte* sender)
 {
     int ret = 0;
-#ifdef WOLFSSL_SHA384
```

Comment:
This code was added because it was needed at some point, Microchip acceleration or something?  Are you saying it's redundant?  Handled elsewhere?

Reply:
Thanks for clarifying.